test/file1-1.tsx

```diff
@@ -0,0 +1,129 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { Collapse } from '@xsite-ui/common.collapse';
+import { Image } from '@xsite-ui/common.image';
+import { CtaButton } from '@xsite-ui/fintech.cta-button';
+import { PoweredByBankrate } from '@xsite-ui/fintech.icons/powered-by-bankrate';
+import { ProductIdentifier } from '@xsite-ui/fintech.product-identifier';
```

Comment:
Importing 'ProductIdentifier' from '@xsite-ui/fintech.product-identifier' but it is not used in the code.